test/integration/app-bundle-integration.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+import { before, describe, test, after } from 'mocha'
+import { readFileSync } from 'fs'
+import { client } from '../helpers'
+
+describe('AppBundle api', function () {
+  let organization
+  let appDefinition
+  let appUpload
+
+  before(async () => {
+    organization = await client()
+      .getOrganizations()
+      .then((response) => response.items[0])
+
+    appDefinition = await organization.createAppDefinition({
+      name: 'Test AppBundle',
+    })
+
+    appUpload = await organization.createAppUpload(readFileSync(`${__dirname}/fixtures/build.zip`))
+  })
+
+  after(async () => {
+    organization.getAppDefinitions().then((response) => {
+      response.items.map((appDefinition) => appDefinition.delete())
+    })
+
+    if (appUpload) {
+      appUpload.delete()
+    }
+  })
+
+  test('Create, get, get all and delete AppBundle', async () => {
+    return appDefinition
+      .createAppBundle({ appUploadId: appUpload.sys.id, comment: 'Test comment' })
+      .then((appBundle) => {
+        expect(appBundle.sys.type).equals('AppBundle', 'type')
+        expect(appBundle.comment).equals('Test comment', 'comment')
+        expect(appBundle.files).to.be.an('array')
```

Comment:
Should we check for the array content to make sure it actually contains something?

Reply:
Yep will add a check to see if we upload an `index.html` file !